commit 35f549ac30d4d1694a9887f3812c56daee46e92a
gpg: Signatur vom Do 20 Jan 2022 10:43:10 CET
gpg:                mittels RSA-Schlüssel D50175CE6FE935BBF28602C5918D8C954C08DB67
gpg: Korrekte Signatur von "Adrian Schroeter (OBS Project Manager) <adrian@suse.de>" [ultimativ]
Author: Adrian Schröter <adrian@suse.de>
Date:   Thu Jan 20 10:42:58 2022 +0100

    compile fix after removal of stb_perlin.h

diff --git a/tests/Makefile b/tests/Makefile
index 1782ea6..1a997af 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -6,7 +6,7 @@ CPPFLAGS = -Wno-write-strings -DSTB_DIVIDE_TEST
 #CFLAGS += -O -fsanitize=address 
 
 all:
-	$(CC) $(INCLUDES) $(CFLAGS) ../stb_vorbis.c test_c_compilation.c test_c_lexer.c test_dxt.c test_easyfont.c test_image.c test_image_write.c test_perlin.c test_sprintf.c test_truetype.c test_voxel.c -lm
-	$(CC) $(INCLUDES) $(CPPFLAGS) -std=c++0x test_cpp_compilation.cpp -lm -lstdc++
+	$(CC) $(INCLUDES) $(CFLAGS) ../stb_vorbis.c test_c_compilation.c test_c_lexer.c test_dxt.c test_easyfont.c test_image.c test_image_write.c test_sprintf.c test_truetype.c test_voxel.c -lm
+#	$(CC) $(INCLUDES) $(CPPFLAGS) -std=c++0x test_cpp_compilation.cpp -lm -lstdc++
 	$(CC) $(INCLUDES) $(CFLAGS) -DIWT_TEST image_write_test.c -lm -o image_write_test
 	$(CC) $(INCLUDES) $(CFLAGS) fuzz_main.c stbi_read_fuzzer.c -lm -o image_fuzzer
diff --git a/tests/stb.dsp b/tests/stb.dsp
index ba13ba1..af16faf 100644
--- a/tests/stb.dsp
+++ b/tests/stb.dsp
@@ -150,10 +150,6 @@ SOURCE=..\stb_malloc.h
 # End Source File
 # Begin Source File
 
-SOURCE=..\stb_perlin.h
-# End Source File
-# Begin Source File
-
 SOURCE=..\stb_rect_pack.h
 # End Source File
 # Begin Source File
@@ -218,10 +214,6 @@ SOURCE=.\test_packer.c
 # End Source File
 # Begin Source File
 
-SOURCE=.\test_perlin.c
-# End Source File
-# Begin Source File
-
 SOURCE=.\test_sprintf.c
 # End Source File
 # Begin Source File
diff --git a/tests/test.sbm b/tests/test.sbm
index 9465605..f151867 100644
--- a/tests/test.sbm
+++ b/tests/test.sbm
@@ -53,7 +53,6 @@ main.c test_dxt.c
 main.c test_easyfont.c
 main.c test_image.c
 main.c test_image_write.c
-main.c test_perlin.c
 main.c test_sprintf.c
 main.c test_vorbis.c ../stb_vorbis.c
 main.c test_voxel.c
diff --git a/tests/test_c_compilation.c b/tests/test_c_compilation.c
index 11f6023..27be5d6 100644
--- a/tests/test_c_compilation.c
+++ b/tests/test_c_compilation.c
@@ -17,7 +17,6 @@
 #include "stb_herringbone_wang_tile.h"
 #include "stb_image.h"
 #include "stb_image_write.h"
-#include "stb_perlin.h"
 #include "stb_c_lexer.h"
 #include "stb_divide.h"
 #include "stb_image_resize.h"
@@ -46,4 +45,4 @@ int quicktest(void)
    char buffer[999];
    stbsp_sprintf(buffer, "test%%test");
    return 0;
-}
\ No newline at end of file
+}
diff --git a/tests/test_cpp_compilation.cpp b/tests/test_cpp_compilation.cpp
index fd8c5b6..43ec5a2 100644
--- a/tests/test_cpp_compilation.cpp
+++ b/tests/test_cpp_compilation.cpp
@@ -6,7 +6,6 @@
 #include "stb_truetype.h"
 #include "stb_image_write.h"
 #include "stb_c_lexer.h"
-#include "stb_perlin.h"
 #include "stb_dxt.h"
 #include "stb_divide.h"
 #include "stb_herringbone_wang_tile.h"
@@ -45,7 +44,6 @@ void my_free(void *) { }
 #include "stb_rect_pack.h"
 #include "stb_truetype.h"
 #include "stb_image_write.h"
-#include "stb_perlin.h"
 #include "stb_dxt.h"
 #include "stb_divide.h"
 #include "stb_herringbone_wang_tile.h"
diff --git a/tests/test_perlin.c b/tests/test_perlin.c
deleted file mode 100644
index de702c9..0000000
--- a/tests/test_perlin.c
+++ /dev/null
@@ -1 +0,0 @@
-#include "stb_perlin.h"
\ No newline at end of file
